app/models/organization.rb

```diff
@@ -29,21 +29,24 @@ class Organization < ApplicationRecord
   belongs_to :parent_organization, class_name: "Organization"
   belongs_to :auto_user, class_name: "User"
 
+  has_many :bike_organizations
+  has_many :bikes, through: :bike_organizations
   has_many :recovered_records, through: :bikes
-  has_many :locations, inverse_of: :organization, dependent: :destroy
+
   has_many :memberships, dependent: :destroy
-  has_many :mail_snippets
   has_many :users, through: :memberships
+
+  has_many :creation_states
+  has_many :created_bikes, through: :creation_states, source: :bike
+
+  has_many :locations, inverse_of: :organization, dependent: :destroy
+  has_many :mail_snippets
```

Comment:
The order of definition seems to matter for these now? This change groups associations so dependencies are defined when dependent associations are defined.